gazebo_usar_core/RadiationSubstance[] radiationSubstances
